src/main/java/com/synopsys/integration/detect/tool/binaryscanner/BinaryUploadOperation.java

```diff
@@ -69,7 +70,11 @@ private void handleUploadError(BinaryUploadStatus status) throws DetectUserFrien
         
         IntegrationException exception = null;
         
-        if (status.getException().isPresent()) {
+        if (status.getStatusCode() == HttpStatus.SC_NOT_FOUND) {
+            // This is a special case until we support only 2024.7 and later BlackDuck's that have
+            // the multipart upload endpoint.
+            exception = new IntegrationException("Submitted file exceeds the file size upload limit.");
```

Comment:
This is for older BlackDuck's where the limit is either 5 or 6 gb. I didn't include the size since Detect won't know the limit until we try the upload.

Reply:
Doh! Of course.